ui/src/components/Secrets.vue

```diff
@@ -429,14 +453,19 @@ async function fetchConfig() {
 }
 
 async function fetchNamespaces(config) {
-  try {
-    const response = await fetch(`${config.api_url}/namespaces`);
-    namespaces.value = await response.json();
-  } catch (error) {
-    setErrorMessage(error);
+  if (import.meta.env.VITE_MOCK_NAMESPACES) {
```

Comment:
Why configure such a mock? We could require e2e to have an backend service running, don't you think?